doc/deployment/deploy-to-AKS-blueegreen-using-jenkins.md

```diff
@@ -0,0 +1,200 @@
+2. Change directory to /src/aks/setup. Run the following kubectl commands
+    
+    Deploy a standard tomcat imgae to the blue and green environments:
+    ```
+    kubectl apply -f  deployment-blue.yml
+    kubectl apply -f  deployment-green.yml
+    ```
+
+    Set up the services for the public end point and the two test end points:
+    ```
+    kubectl apply -f  service-green.yml
+    kubectl apply -f  test-endpoint-blue.yml
+    kubectl apply -f  test-endpoint-green.yml
+    ```
+
+3. Update the public and test end points DNS names. When AKS is created, an [additional resource group]( https://github.com/Azure/AKS/issues/3) is created. Look for resource group: MC_\<your-resource-group-name\>_\<your-kubernetes-cluster-name\>_\<your-location\>.
+
+    Locate the public ip's in the resource group
+    ![Public IP](images/publicip.png)
+
+    For each of the services, find the external IP address by running:
+    
+    ```
+    kubectl get service todoapp-service
+    ```
+    
+    Update the DNS name for the corresponding IP address:
+
+    ```
+    az network public-ip update --dns-name aks-todoapp --ids /subscriptions/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/resourceGroups/MC_<resourcegroup>_<aks>_<location>/providers/Microsoft.Network/publicIPAddresses/kubernetes-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+    ```
+
+    Repeat for `todoapp-test-blue` and `todoapp-test-green`:
+    ```
+    az network public-ip update --dns-name todoapp-blue --ids /subscriptions/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/resourceGroups/MC_<resourcegroup>_<aks>_<location>/providers/Microsoft.Network/publicIPAddresses/kubernetes-BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB
+
+    az network public-ip update --dns-name todoapp-green --ids /subscriptions/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/resourceGroups/MC_<resourcegroup>_<aks>_<location>/providers/Microsoft.Network/publicIPAddresses/kubernetes-CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+    ```
+    Note that the dns name needs to be unique in your subscription.
+
+    Now, you can visit your app by navigating to e.g.,  http://aks-todoapp.<your-location>.cloudapp.azure.com. If you see the following, you have succeeded in setting up your AKS cluster.  
+    ![Tomcat](images/tomcat.png)
+
+### Create Azure Container Registry
+
+1. Run below command to create an Azure Container Registry.
+After creation, use `login server` as Docker registry URL in the next section.
+
+   ```bash
+   az acr create -n <your-registry-name> -g <your-resource-group-name>
+   ```
+
+1. Run below command to show your Azure Container Registry credentials.
+You will use Docker registry username and password in the next section.
+
+    ```bash
+    az acr credential show -n <your-registry-name>
+    ```
+
+## Prepare Jenkins server
+1. Deploy a Jenkins Master on Azure [https://aka.ms/jenkins-on-azure]
+
+1. Connect to the server with SSH and install the build tools on the server where you will run your build:
+   
+   ```
+   sudo apt-get install git maven docker.io
+   ```
+

```

Comment:
As discussed in email, let's update Docker install with the instructions from Docker documentation.